### ISSUE TITLE
This analysis terminated unexpectedly.  Error in dyn.load(file, DLLpath = DLLpath, ...): unable to load shared object '/Applications/JASP.app/Contents/Frameworks/R.framework/Versions/4.2/Resources/library/Matrix/libs/Matrix.so':   dlopen(/Applications/JASP.app/Contents/Frameworks/R.framework/Versions/4.2/Resources/library/Matrix/libs/Matrix.so, 6): Library not loaded: @executable_path/../Frameworks/R.framework/Versions/4.2/Resources/opt/local/gfortran/lib/libgfortran.dylib   Referenced from: /Applications/JASP.app/Contents/Frameworks/R.framework/Versions/4.2/Resources/library/Matrix/libs/Matrix.so   Reason: image not found[Bug]: 

### ISSUE THREAD

**Issue by huwhushw** [jasp-stats/jasp-issues#2408]
### JASP Version

0.18.1

### JASP Module

ANOVA

### What OS are you seeing the problem on?

macOS Silicon, ChromeOS

### Bug Description

This analysis terminated unexpectedly.

### Expected Behaviour

Fix it for me asap! 

### Steps to Reproduce

1.I already tried multiple times. 
2.
3.
...

### Final Checklist

- [X] I have included a screenshot showcasing the issue, if possible.
- [X] I have included a JASP file (zipped) or data file that causes the crash/bug, if applicable.
- [X] I have accurately described the bug, and steps to reproduce it.

**Comment by huwhushw** [comment]
![截屏2023-11-13 下午1 00 53](https://github.com/jasp-stats/jasp-issues/assets/150720730/da6e5ec7-d3f3-4fff-96f3-7522a5034802)

**Comment by huwhushw** [comment]
Can you take a look and fix it for me asap?

**Comment by JohnnyDoorn** [comment]
hi @huwhushw ,

Can you try [redownloading and reinstalling the latest version of JASP](https://jasp-stats.org/download/)? I think your issue is fixed in the latest release. 

Cheers,
Johnny

**Comment by huwhushw** [comment]
Yes, but I believe my MacBook Air's version is too old. The newest version of JSAP won't accept my Catalina version.
________________________________
发件人: Johnny van Doorn ***@***.***>
发送时间: 2023年11月13日 13:46
收件人: jasp-stats/jasp-issues ***@***.***>
抄送: huwhushw ***@***.***>; Mention ***@***.***>
主题: Re: [jasp-stats/jasp-issues] This analysis terminated unexpectedly. Error in dyn.load(file, DLLpath = DLLpath, ...): unable to load shared object '/Applications/JASP.app/Contents/Frameworks/R.framework/Versions/4.2/Resources/library/Matrix/libs/Matrix...

hi @huwhushw , Can you try redownloading and reinstalling the latest version of JASP? I think your issue is fixed in the latest release. Cheers, Johnny ― Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving


hi @huwhushw<https://urldefense.com/v3/__https://github.com/huwhushw__;!!KGKeukY!0iQogP2mfJQ6ZmMm_IwoRcxaUGs7tQfQnuADz3tkkRhEXRCQNmyuyY8P_H8O2hAhvRTNtfwFGKl4l5IL8xRvSWHwY0dT$> ,

Can you try redownloading and reinstalling the latest version of JASP<https://urldefense.com/v3/__https://jasp-stats.org/download/__;!!KGKeukY!0iQogP2mfJQ6ZmMm_IwoRcxaUGs7tQfQnuADz3tkkRhEXRCQNmyuyY8P_H8O2hAhvRTNtfwFGKl4l5IL8xRvSTrY36y9$>? I think your issue is fixed in the latest release.

Cheers,
Johnny

―

**Comment by JohnnyDoorn** [comment]
Is there a specific error you get? 
Also tagging @boutinb because he might be of better help.

**Comment by boutinb** [comment]
Hi @huwhushw We had this problem with MacOS Intel. Are you sure you have MacOS Silicon (check it in the "About this mac")? If you have MacOS Intel, then you should try this JASP version https://static.jasp-stats.org//JASP-0.18.1.0-macOS-x86_64_new2.dmg
If your Mac OS is too old for this, then you should try an older JASP version here https://jasp-stats.org/previous-versions Mac OS Catalina is only officially supported in JASP 0.17.3. Apple does not support Catalina anymore, so we cant support it anymore.
